src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = 0;
+  const DAY_COST = 40;
+  let discount = 0;
+
+  for (let n = 1; n <= days; n++) {
```

Comment:
Using a `for` loop here is unnecessary and inefficient. The total cost can be calculated directly from the number of days without iterating through each day.